components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -165,7 +165,9 @@ tweaking their $case/namelist_scream.xml file.
     <!-- P3 microphysics -->
     <p3 inherit="physics_proc_base">
       <do_prescribed_ccn>true</do_prescribed_ccn>
+      <do_prescribed_ccn COMPSET=".*SCREAM*noSPA*">false</do_prescribed_ccn>
```

Comment:
The regex for the wildcard should go as

COMPSET=".*SCREAM.*noSPA" or COMPSET=".*SCREAM.*noAero"

The original form, SCREAM*noSPA would try to match a patterns with 0 or more 'M' between 'SCREAM' and 'noSPA', which is not intended. The new form is to allow any number of characters in-between.